chall03/vkuikka.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+	"strconv"
+	"io/ioutil"
+)
+
+func	getData(addr string) string {
+	fmt.Println(addr)
+	response, err := http.Get(addr)
+	if err != nil {
+		return ("error")
+	}
+	print(string(response.StatusCode) + response.Status + "\n")
+	responseData, err := ioutil.ReadAll(response.Body)
+	if err != nil {
+		return ("error")
```

Comment:
in Go you can return 2 variables (or more)

This function should be re-written to have (string, error) return variables then here you could have written:

```go
return nil, errors.New("can't read the body")
```